objective.py

```diff
@@ -32,7 +32,7 @@ class Objective(BaseObjective):
     # solvers or datasets should be declared in Dataset or Solver (see
     # simulated.py and python-gd.py).
     # Example syntax: requirements = ['numpy', 'pip:jax', 'pytorch:pytorch']
-    requirements = ["pytorch", "numpy", "deepinv"]
+    requirements = ["pytorch", "numpy", "deepinv", "pyiqa"]
```

Comment:
Why do you need this?

Reply:
It's a package requirement for methods in deepinv that is not installed as a dependency.